cluster-api-gcp/deploy/deploy_helper.go

```diff
@@ -293,3 +299,21 @@ func (d *deployer) waitForApiserver(master string, timeout time.Duration) error
 	}
 	return err
 }
+
+// Make sure the default service account in kube-system namespace exists.
+func (d *deployer) waitForServiceAccount(timeout time.Duration) error {
+	client, err := apiutil.NewKubernetesClient(d.configPath)
+	if err != nil {
+		return err
+	}
+	startTime := time.Now()
+
+	for time.Now().Sub(startTime) < timeout {
+		_, err := client.CoreV1().ServiceAccounts("kube-system").Get("default-foo", metav1.GetOptions{})
+		if err == nil {
+			return nil
+		}
+		time.Sleep(1 * time.Second)
```

Comment:
Consider adding the sleep time as a constant.  See the line below as a reference:
https://github.com/karan/kube-deploy/blob/760c8e423e826f61afb7b7748bdb1761c8a30bed/cluster-api-gcp/deploy/deploy_helper.go#L39

Reply:
I now use the already defined SleepSecondsPerAttempt.